docs/api/rsc/RSCHydratedRouter.md

```diff
@@ -11,6 +11,52 @@ unstable: true
 
 Hydrates a server rendered `RSCPayload` in the browser.
 
+```tsx filename=entry.browser.tsx lines=[8,33-39]
+"use client-entry";
+
+import * as React from "react";
+import { hydrateRoot } from "react-dom/client";
+import {
+  unstable_createCallServer as createCallServer,
+  unstable_getRSCStream as getRSCStream,
+  unstable_RSCHydratedRouter as RSCHydratedRouter,
+} from "react-router";
+import type { unstable_RSCPayload as RSCPayload } from "react-router";
+import {
+  createFromReadableStream,
+  createTemporaryReferenceSet,
+  encodeReply,
+  setServerCallback,
+} from "react-server-dom-parcel/client";
+
+setServerCallback(
+  createCallServer({
+    createFromReadableStream,
+    createTemporaryReferenceSet,
+    encodeReply,
+  })
+);
```

Comment:
Small suggestion: maybe remove the extraneous code in these API docs to keep them focused? The code highlighting definitely helps, but I'm just thinking it might better to strictly show how to setup these APIs

Reply:
yeah I went back and forth on that originally since some of the functions aren't neccesarily clear where they come from but I think you're right that streamlined docs are better for consumption and they can go look at the template if they need the full picture. 

Updated and streamlined based on the usages in https://github.com/remix-run/react-router-templates/blob/brooks/rsc-template-updates/unstable_rsc-parcel